src/5e-SRD-Features.json

```diff
@@ -142,6 +255,27 @@
     "desc": [
       "Starting at 5th level, your speed increases by 10 feet while you aren't wearing heavy armor."
     ],
+    "feature_specific": {
+      "speed": {
+        "type": "sum",
+        "values": [
+          {
+            "type": "integer",
+            "from": "base"
+          },
+          {
+            "type": "integer",
+            "value": 10
+          }
+        ],
+        "prerequisites": [
+          {
+            "type": "boolean",
+            "from": "heavy armor"
```

Comment:
Instead of `"heavy armor"`, what about something like `equipped.heavy_armor`?

Reply:
I'd go with `equipped.armor.heavy`, since there is no `heavy_armor` label when it comes to equipment. Item can can have category `armor', and then a prop specifying the heaviness - `armor_category`

Reply:
Yeah, that works